CHANGELOG.md

```diff
@@ -14,30 +14,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Maintenance
 ### Refactoring
 
-## [Unreleased 2.x](https://github.com/opensearch-project/neural-search/compare/2.18...2.x)
+## [Unreleased 2.x](https://github.com/opensearch-project/neural-search/compare/2.19...2.x)
```

Comment:
Could you remove
```
- Fix for nested field missing sub embedding field in text embedding processor ([#913](https://github.com/opensearch-project/neural-search/pull/913))
```

in line 11?

Reply:
Sure. Thanks for the catch. 

Looks like this is already released in [2.18](https://github.com/opensearch-project/neural-search/commits/2.18.0.0/) as per https://github.com/opensearch-project/neural-search/pull/913. Should I also add it to [2.18 release notes](https://github.com/opensearch-project/neural-search/blob/main/release-notes/opensearch-neural-search.release-notes-2.18.0.0.md) as well?

Reply:
I don't think we're changing already released notes

Reply:
I think we can change if release note is not correct. However, as that is minor bug fix, let's ignore this time?

Reply:
Okay. I will remove it from 3.0 section, and not add it to 2.18 release notes.